android/app/src/main/java/org/unicef/ecar/bebbo/ExactAlarmPackage.java

```diff
@@ -0,0 +1,24 @@
+package org.unicef.ecar.bebbo;
+import com.facebook.react.ReactPackage;
+import com.facebook.react.bridge.NativeModule;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.uimanager.ViewManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ExactAlarmPackage implements ReactPackage {
+
+    @Override
+    public List<ViewManager> createViewManagers(ReactApplicationContext reactContext) {
+        return Collections.emptyList();
+    }
+
+    @Override
+    public List<NativeModule> createNativeModules(ReactApplicationContext reactContext) {
+        List<NativeModule> modules = new ArrayList<>();
+        modules.add(new ExactAlarmModule(reactContext));
+        return modules;
+    }
+}
```

Comment:
Add some comments to the class and methods.

Reply:
added appropriate comment